www-apps/redmine/redmine-4.0.5.ebuild

```diff
@@ -0,0 +1,224 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+USE_RUBY="ruby24"
+inherit eutils depend.apache ruby-ng
+
+DESCRIPTION="Flexible project management web application using the Ruby on Rails framework"
+HOMEPAGE="https://www.redmine.org/"
+SRC_URI="https://www.redmine.org/releases/${P}.tar.gz"
+
+KEYWORDS="~amd64"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="imagemagick fastcgi ldap markdown mysql passenger postgres sqlite"
+
+ruby_add_rdepend "
+	imagemagick? ( dev-ruby/rmagick:0 )
+	ldap? ( dev-ruby/ruby-net-ldap )
+	markdown? ( >=dev-ruby/redcarpet-3.4.0 )
+	mysql? ( >=dev-ruby/mysql2-0.5.0:0.5 )
+	postgres? ( >=dev-ruby/pg-1.1.4:1 )
+	sqlite? ( >=dev-ruby/sqlite3-1.3.12 )
+	dev-ruby/actionpack-xml_parser:*
+	dev-ruby/i18n:0.7
+	>=dev-ruby/mail-2.7.1
+	dev-ruby/mimemagic
+	>=dev-ruby/mini_mime-1.0.1
+	>=dev-ruby/nokogiri-1.10.0
+	dev-ruby/rails:5.2
+	>=dev-ruby/rbpdf-1.19.6
+	dev-ruby/request_store:1.0.5
+	>=dev-ruby/roadie-rails-1.3.0
+	>=dev-ruby/rouge-3.6.0
+	>=dev-ruby/ruby-openid-2.9.2
+	dev-ruby/rack-openid
+"
+
+RDEPEND="
+	acct-group/redmine
+	acct-user/redmine
+"
+
+REDMINE_DIR="/var/lib/${PN}"
+
+all_ruby_prepare() {
+	rm -r log files/delete.me .github || die
+
+	# bug #406605
+	rm .{git,hg}ignore || die
+
+	# newenvd not working here
+	cat > "${T}/50${PN}" <<-EOF || die
+		CONFIG_PROTECT="${EROOT%/}${REDMINE_DIR}/config"
+		CONFIG_PROTECT_MASK="${EROOT%/}${REDMINE_DIR}/config/locales ${EROOT%/}${REDMINE_DIR}/config/settings.yml"
+	EOF
```

Comment:
This should work in `all_ruby_install` because it's src_install function. Similar to fowners and fperms.